src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ChefExtensionTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
+{
+    public class ChefExtensionTests
+    {
+        public ChefExtensionTests(Xunit.Abstractions.ITestOutputHelper output)
+        {
+            ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSetChefExtensionBasic()
+        {
+            ComputeTestController.NewInstance.RunPsTest("Test-SetChefExtensionBasic");
```

Comment:
@markcowl , @shahabhijeet 
-- Still facing this error while running the tests:
```
The term 'Test-SetChefExtensionBasic' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
Basically `Test-SetChefExtensionBasic` is the method we defined in the PS script which is supposed to be called from the C# test script. The method is not being recognized.
-- While debugging found that the powershell script is being loaded automatically here: https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Compute/Commands.Compute.Test/Common/ComputeTestController.cs#L146. So we don't have to add the path anywhere.
But `Events` object is null here: https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs#L367. After this the error is thrown.
-- Tried to run the existing tests too like `AvailabilitySetTests` and `DscExtensionTests`, but the error was raised at the same point mentioned above.